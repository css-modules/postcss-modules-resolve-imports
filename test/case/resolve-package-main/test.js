'use strict';

const setup = require('../../setup');

test('resolve-package-main', () => {
  const {resulting, exports: tokens} = setup(
    'local-by-default',
    'extract-imports',
    'scope',
    'self'
  )(__dirname);

  expect(resulting).toMatchSnapshot();
  expect(tokens).toMatchSnapshot();
});
