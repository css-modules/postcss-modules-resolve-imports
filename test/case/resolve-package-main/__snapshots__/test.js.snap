// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resolve-package-main 1`] = `
"/* imported from button/my-awesome-button.css */

._my-awesome-button_button
{
  /*common button styles*/
}

/* imported from source.css */

._source_continueButton
{
  color: green;
}
"
`;

exports[`resolve-package-main 2`] = `
Object {
  "continueButton": "_source_continueButton _my-awesome-button_button",
}
`;
